index.js

```diff
@@ -9,6 +10,10 @@ const port = 3001;
 app.listen(port, () => console.log(`Steam reviews service. listening at http://localhost:${port}`));
 app.use(express.static('./client/dist'));
 
+app.get('/game/:gameId', (req, res) => {
```

Comment:
What's your rationale behind sending the same html file for each different gameId route? I understand that the file would be able to read the gameid from the url parameters and render approrpriately based off that, but doesn't express.static already server your index.html at root route? Just curious - not requesting changes for this

Reply:
I was running into issues serving the index.html WITH the extra slash param. 

From just the base static service, it was returning "Cannot GET /game/41", which I assumed meant that it was looking for "client/dist/games", but even moving the index in there wasn't helping, so it was a bit of an act of desperation. 

I don't so much need the :gameId in there (the server obviously isn't using it) as much as that's the pattern I want to match to serve the root's index.hrml.

If there's a better way to achieve this, I would love to do it that way!

Reply:
I'm really not sure what the problem would be then, without cloning the whole repo & reenacting the behaviors that lead to this. I agree with your other reply about Help Desk though - if time allows, maybe someone from there would have a better handle on this. Good luck!